low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -181,7 +183,7 @@ class SSPLDevDeploy:
                print(f"ERROR: Unable to download {pkg_name}.")
             cmd = "yum install -y %s%s" % (build_url, pkg[0])
             returncode = os.system(cmd)
-            if returncode != 0:
+            if not (returncode==0 or returncode==256):
```

Comment:
what is this special case ?

Reply:
1. If found cortx-py-utils installed already when installing it from web URL, returncode is 256 stating that package already installed(Error).
2. If not installed already but installing now, returncode is 0 stating that package already installed.
`
>>> cmd
'yum install -y http://cortx-storage.colo.seagate.com/releases/eos/github/main/centos-7.8.2003/cortx-utils_99/prod/cortx_iso/cortx-py-utils-1.0.0-99_23eb199.noarch.rpm'
>>>
>>> returncode = os.system(cmd)
Loaded plugins: fastestmirror, product-id, search-disabled-repos, subscription-manager
cortx-py-utils-1.0.0-99_23eb199.noarch.rpm                                                                 | 212 kB  00:00:00
Examining /var/tmp/yum-root-wLc7tI/cortx-py-utils-1.0.0-99_23eb199.noarch.rpm: cortx-py-utils-1.0.0-99_23eb199.noarch
/var/tmp/yum-root-wLc7tI/cortx-py-utils-1.0.0-99_23eb199.noarch.rpm: does not update installed package.
Error: Nothing to do
>>>
>>> returncode
256
>>> cmd = "yum install -y cortx-py-utils"
>>> returncode = os.system(cmd)
Loaded plugins: fastestmirror, product-id, search-disabled-repos, subscription-manager
Package matching cortx-py-utils-1.0.0-83_8b3c613.noarch already installed. Checking for update.
Nothing to do
>>>
>>> returncode
0
`